### ISSUE TITLE
[Bug]: Incorrect `count` results lead to occasional failures in bvt testing.

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#9176]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):  sry, not sure
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="725" alt="image" src="https://user-images.githubusercontent.com/77312370/234450085-c844a1a8-c34a-4337-b0d3-d2047222746e.png">

https://github.com/matrixorigin/matrixone/actions/runs/4794727705/jobs/8529534026

**Comment by nnsgmsone** [comment]
I found that there is a situation that after a table is successfully subscribed, dn will think that the table is not successfully subscribed, and then ignore the logs of this table, thus leading to incorrect data, help @triump2020 :
```
2023/04/25 12:46:11.509597 +0000 INFO frontend/util.go:400 query trace status {"connection_id": 103, "statement": "insert into t select * from generate_series(1,200000) g", "status": "success", "span": {"trace_id": "274be702-e367-11ed-960f-000d3aee994d", "kind": "statement"}, "session_info": "connectionId 103|127.0.0.1:56270|{account sys:dump:moadmin -- 0:1:0}|f03cd42c-e366-11ed-960f-000d3aee994d"}
2023/04/25 12:46:11.510530 +0000 INFO frontend/util.go:400 query trace {"connection_id": 103, "query": "select count(*) from t;", "session_info": "connectionId 103|127.0.0.1:56270|{account sys:dump:moadmin -- 0:1:0}|f03cd42c-e366-11ed-960f-000d3aee994d"}
2023/04/25 12:46:11.512085 +0000 INFO disttae/logtailPushModel.go:176 send subscribe tbl[db: 273673, tbl: 273703] request succeed
2023/04/25 12:46:11.512172 +0000 INFO dn-service.logtail-push-service service/server.go:383 handle subscription asynchronously {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "table": "db_id:273673 tb_id:273703 "}
2023/04/25 12:46:11.561967 +0000 INFO logtail/handle.go:566 [logtail] table data [Insert] 273703-t: [BatchName=data]
(attr=__mo_rowid)[200000]: 282f8643-e367-11ed-960f-000d3aee994d-3-0-0,282f8643-e367-11ed-960f-000d3aee994d-3-0-1,282f8643-e367-11ed-960f-000d3aee994d-3-0-2
(attr=commit_time)[200000]: 1682426771392013103-0,1682426771392013103-0,1682426771392013103-0
(attr=a)[200000]: 24577,24578,24579

2023/04/25 12:46:11.562061 +0000 INFO dn-service.logtail-push-service service/session.go:412 send subscription response {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "table": "db_id:273673 tb_id:273703 ", "To": "PhysicalTime:1682426771510703006 "}
2023/04/25 12:46:11.819131 +0000 INFO gc/manager.go:93 processing job=clean-transfer-table, inv=1m0s
2023/04/25 12:46:11.819518 +0000 INFO gc/manager.go:93 processing job=checkpoint-gc, inv=1m0s
2023/04/25 12:46:11.819533 +0000 INFO gc/manager.go:93 processing job=logtail-gc, inv=1m0s
2023/04/25 12:46:11.824991 +0000 INFO jobs/compactblk.go:137 [Start] {"operation": "[4688]compact", "operand": "[A]BLK[<0:273703-282f8643-e367-11ed-960f-000d3aee994d-23-0:0-0>]"}
2023/04/25 12:46:11.832025 +0000 INFO blockio/writer.go:134 [WriteEnd] {"operation": "name: 282f8643-e367-11ed-960f-000d3aee994d_00025, block count: 1, size: 32934"}
2023/04/25 12:46:11.832225 +0000 INFO jobs/compactblk.go:137 [Start] {"operation": "[4690]compact", "operand": "[A]BLK[<0:273703-282f8643-e367-11ed-960f-000d3aee994d-22-0:0-0>]"}
2023/04/25 12:46:11.833475 +0000 INFO jobs/compactblk.go:137 [Start] {"operation": "[4692]compact", "operand": "[A]BLK[<0:273703-282f8643-e367-11ed-960f-000d3aee994d-12-0:0-0>]"}
2023/04/25 12:46:11.834639 +0000 INFO jobs/compactblk.go:137 [Start] {"operation": "[4694]compact", "operand": "[A]BLK[<0:273703-282f8643-e367-11ed-960f-000d3aee994d-15-0:0-0>]"}
2023/04/25 12:46:11.835777 +0000 INFO jobs/compactblk.go:137 [Start] {"operation": "[4696]compact", "operand": "[A]BLK[<0:273703-282f8643-e367-11ed-960f-000d3aee994d-6-0:0-0>]"}
2023/04/25 12:46:12.081214 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081251 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081259 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081295 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081309 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081344 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081352 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081389 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081398 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081434 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081446 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081484 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
2023/04/25 12:46:12.081492 +0000 INFO dn-service.logtail-push-service service/session.go:334 table not subscribed, filter out {"uuid": "dd4dccb4-4d3c-41f8-b482-5251dc7a41bf", "server-id": "edf2a3b5-b877-4848-84f6-7c8424a56308", "stream-id": 1, "id": "db_id:273673 tb_id:273703 ", "checkpoint": ""}
```

**Comment by nnsgmsone** [comment]
i take it

**Comment by nnsgmsone** [comment]
Some cases are the same as #9177, the case of delete still needs to be checked.

**Comment by nnsgmsone** [comment]
wait for fix the s3 delete

**Comment by triump2020** [comment]
No progress

**Comment by aressu1985** [comment]
testing

**Comment by aressu1985** [comment]
testing

**Comment by aressu1985** [comment]
fixed:fad1eaeace72fe6bacac673181020420a21d48ca
